browser/ui/brave_scheme_utils.cc

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/browser/ui/brave_scheme_utils.h"
+
+#include "base/strings/string_util.h"
+
+namespace brave_utils {
+constexpr char16_t kChromeSchema16[] = u"chrome://";
+constexpr char16_t kBraveSchema16[] = u"brave://";
```

Comment:
wait, these should be either in anonymous namespace or just be a part of the function. Right now these variables can be declared as `extern ...` and be used anywhere, and also they can conflict with a symbol with the same name elsewhere.

Reply:
@goodov Initially I thought of having this header in header like `inline constexpr char16_t kChromeSchema16[] = u"chrome://";` and use it elsewhere when needed from here instead of redefining in places,  Whats your thought on to go with this?

Reply:
not sure if these schemes in `char16_t` will be used widely tbh. I'd just keep them here in anonymous ns.

Reply:
Okay! For now I'll keep it in the anonymous namespace.